File: PRAK-4/PRAK405-2410817110011-MUHAMMADAZRIELAKBAR.py
pengali, kelipatan = map(int, input().split())
total = 0

for i in range (1 , pengali + 1, 1):
  jumlah = 0
  print("(", end="")
  for j in range (i, 0, -1) :
    nilai = j * kelipatan
    print(f"{j} * {kelipatan}", end="")
    jumlah = jumlah + nilai
    if j > 1 :print(") + (", end="")
  print(f") = {jumlah}")
  total = total + jumlah
print(f"{total}")

File: PRAK-4/PRAK405-2410817110011-MUHAMMADAZRIELAKBAR.c
#include <stdio.h>
int main()
{
  int pengali, kelipatan;
  int total = 0;
  scanf("%d %d", &pengali, &kelipatan);
  for (int i = 1; i <= pengali; i++)
  {
    int jumlah = 0;
    printf("(");
    for (int j = i; j >= 1; j--)
    {
      int nilai = j * kelipatan;
      printf("%d * %d", j, kelipatan);
      jumlah = jumlah + nilai;
      if (j > 1)
        printf(") + (");
    }
    printf(") = %d \n", jumlah);
    total += jumlah;
  }
  printf("%d \n", total);
}